src/database/sequlize-handler/migrations/00_jobs_proxy_and_debug.js

```diff
@@ -0,0 +1,18 @@
+const Sequelize = require('sequelize');
+
+module.exports.up = async (query, DataTypes) => {
+    await query.addColumn(
+        'jobs', 'proxy_url',
+        Sequelize.DataTypes.STRING);
+
+    await query.addColumn(
+        'jobs', 'debug',
+        Sequelize.DataTypes.STRING);
+};
+
+module.exports.down = async (query, DataTypes) => {
+    await query.sequelize.query([
```

Comment:
is there a reason the down() func is in the form of a raw query? will this work for all databases?

Reply:
changed